presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -607,6 +608,11 @@ public LocalDynamicFiltersCollector getDynamicFiltersCollector()
             return dynamicFiltersCollector;
         }
 
+        public void addDynamicFilter(Map<String, Domain> dynamicTupleDomain)
```

Comment:
this method can be private. Move below usage (`createDynamicFilter`) or just inline it

Reply:
Made it private, it's part of `LocalExecutionPlanContext` inner class so not sure if we want to move it.